### PR TITLE
Version 0.2.0 release (without new services)

### PR DESCRIPTION
**Pull request by fboucquez**
Hi @rg911 @segfaultxavi 

I'm planning to do a 0.2.0 release without new services. The faucet, wallet, explorer services are ready, just that I cannot release with alpha dependencies. I would need to wait for them to release first. If we do some Travis improvements, we can release more often when there is a not symbol wide solution release.

This PR is to release 0.2.0. 

Note that I did a test release of 0.1.2 to try the new Travis release process. It went well, just want to officialize the new changes as version 0.2.0. (I will remove 0.1.2 from releases)

A few small fixes:

- Added nemesis mosaic repeat:0
- Generating Sink addresses when no provided
- Upgraded version
- Fixed healthCheck.ts description
- Revisited e2e test to run in a cron job

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,18 +4,21 @@ All notable changes to this project will be documented in this file.
...
 ---|---|---
-Symbol Bootstrap | v0.1.2 | [symbol-bootstrap](https://www.npmjs.com/package/symbol-bootstrap)
+Symbol Bootstrap | v0.2.0 | [symbol-bootstrap](https://www.npmjs.com/package/symbol-bootstrap)
 
 - **[BREAKING CHANGE]** Target folder structure has been changed for scalability. The old target folder needs to be dropped when running this version. Backup the target folder if you need to keep your data!
 - **New Command:** `symbol-bootstrap resetData` cleans the peer data and database without dropping the generated configuration.
 - **New Command:** `symbol-bootstrap healthCheck` tests if the docker compose network is running locally. `--healthCheck` param is allowed in `start` and `run` commands.
 - Allowed `repeat` on a node, a database or a gateway to instantiate them multiple times. This enables you to create large network configurations quickly.
+- Allowed `repeat` in nemesis's mosaic list. Harvest currency can be removed with `repeat:0`.
+- Removed preset `light`. Not it's an assembly for the bootstrap preset `symbol-bootstrap -p bootstrap -a light`.
```
Comment (segfaultxavi) [inline CHANGELOG.md:20]:
```suggestion
- Allowed `repeat` in the nemesis block's mosaic list. Harvest currency can be removed with `repeat:0`.
- Removed preset `light`. Now it's an assembly for the bootstrap preset: `symbol-bootstrap -p bootstrap -a light`.
```

**Review by segfaultxavi — CHANGES_REQUESTED** [review]
Minor typos.

**Comment by segfaultxavi** [comment]
I'll update the [Using symbol-bootstrap](https://docs.symbolplatform.com/guides/network/using-symbol-bootstrap.html) docs to remove the `light` preset once this lands.

**Comment by fboucquez** [comment]
> I'll update the [Using symbol-bootstrap](https://docs.symbolplatform.com/guides/network/using-symbol-bootstrap.html) docs to turn the `light` preset into an assembly once this lands.

Thank Xavi, with the new repeat, `light` is just bootstrap preset with some services turned off. Easier to maintain and it showcases how `repeat` could work

**Review by segfaultxavi — APPROVED** [review]
👍

**Review by rg911 — APPROVED** [review]

**Pull request merged**